tests/framework/extensions/hardening/rke2/harden_nodes.go

```diff
@@ -43,9 +43,8 @@ func HardeningNodes(client *rancher.Client, hardened bool, nodes []*nodes.Node,
 
 func PostHardeningConfig(client *rancher.Client, hardened bool, nodes []*nodes.Node, nodeRoles []string) error {
 	for key, node := range nodes {
-
 		if strings.Contains(nodeRoles[key], "--controlplane") {
-			dir_local := "/Users/anupamaupadhyayula/go/src/github.com/rancher/rancher/tests/framework/extensions/hardening/rke2"
+			dir_local := "/go/src/github.com/rancher/rancher/tests/framework/extensions/hardening/rke2"
```

Comment:
I know this wasn't you, and this slipped when I reviewed, so I am so sorry. First `dir_local` should be `dirLocal`, a little python slipped. Also could we do this instead to get the file path, because this won't work for everything just in our `Dockerfile.validation` file.

```
user, err := user.Current()
if err != nil {
	return nil, err
}

keyPath = filepath.Join(user.HomeDir, "go/src/github.com/rancher/rancher/tests/framework/extensions/hardening/rke2")
```
the import is `"os/user"`

Reply:
Resolved.